README.md

```diff
@@ -29,3 +29,16 @@ Follow the instructions, and use the created user to access the admin area.
  1 - Run `gem install mailcatcher`
  2 - Run `mailcatcher`
  3 - Go to `http://localhost:1080/`
+
```

Comment:
I'm not taking about the Rails app configuration, but the configuration required no AWS. Which options the SQS queue should be configured there, retries, timeouts, etc.